index.html

```diff
@@ -186,6 +186,109 @@ <h2 class="card-header">Professional Art <br> Printing Data</h2>
+            <h1  class="primary-text">about me</h1>
+        </div>
+        <div>
+            <h2 class="description">
+            Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </h2>
+        </div>
+        <button class="main-btn no_wrap">Get My Resume</button>
+        <hr class="hl-white">
+    </section>
+    <section class="skills-section">
+        <div class="item">
+            <div class="skill-header">
+                <div class="rectangle prism"></div>
+                <h3 class="skill-title">Language</h3>
+            </div>
+            <ul class="skills">
+                <li class="skill">Javascript</li>
+                <li class="skill">Ruby</li>
+                <li class="skill">Html</li>
+                <li class="skill">Css</li>
+            </ul>
+        </div>
+        <!-- second -->
+        <div class="item">
+            <div class="skill-header">
+                <div class="rectangle"></div>
+                <h3 class="skill-title">Frameworks</h3>
+            </div>
+            <ul class="skills">
+                <li class="skill">Bootstrap</li>
+                <li class="skill">Ruby on Rails</li>
+                <li class="skill">RSPec</li>
+                <li class="skill">CapyBara</li>
+                <li class="skill">Selenium</li>
+            </ul>
+        </div>
+        <!-- third -->
+        <div class="item">
+            <div class="skill-header">
+                <div class="rectangle circle"></div>
+                <h3 class="skill-title">Skills</h3>
+            </div>
+            <ul class="skills">
+                <li class="skill">Codekit</li>
+                <li class="skill">Github</li>
+                <li class="skill">Codepen</li>
+                <li class="skill">Gitlab</li>
+                <li class="skill">Terminal</li>
+            </ul>
+        </div>
+    </section>
+    <section class="form-section">
+        <h1 class="form-intro">
+            I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </h1>
+        <form action="" class="form-container">
+            <div class="input-field">
+                <input type="text" class="name-input box" placeholder="Full Name" maxlength="30" required >
+            </div>
+            <div class="input-field">
+                <input type="email" name="email" class="email-input box" placeholder="email address" required>
+            </div>
+            <div class="input-field">
+                <textarea class="box-text" name="comment" id="comment-input" cols="30" rows="10" placeholder="Enter text here" maxlength="500" aria-placeholder="Enter text area ..." required></textarea>
+            </div>
+            <button class="main-btn center">Get in touch</button>
+        </form>
```

Comment:
- [ ] The form looks great, but unfortunately, it is not working as expected because it is not linked with the Formspree service. Kindly follow the 'Get started' section of the [Formspree](https://formspree.io/html/) website and make sure your form is working. Remember to add a `type` to your button at the end of the form.
 - [OPTIONAL] The heading of this section is not the main heading of the page and should respect the hierarchy of the headings. It would be better to use a different `h` tag following the hierarchy. Kindly take a look at [this article](https://www.reliablesoft.net/h1-tag/) for more information